apps/web/components/dialog/EditLocationDialog.tsx

```diff
@@ -242,91 +226,45 @@ export const EditLocationDialog = (props: ISetLocationDialog) => {
                 {t("edit_location")}
               </h3>
-              {!booking && (
-                <p className="text-default text-sm">
-                  <Trans i18nKey="cant_find_the_right_conferencing_app_visit_our_app_store">
-                    Can&apos;t find the right conferencing app? Visit our
-                    <Link
-                      className="cursor-pointer text-blue-500 underline"
-                      href="/apps/categories/conferencing">
-                      App Store
-                    </Link>
-                    .
-                  </Trans>
-                </p>
-              )}
             </div>
             <div className="mt-3 text-center sm:mt-0 sm:text-left" />
 
-            {booking && (
-              <>
-                <p className="text-emphasis mb-2 ml-1 mt-6 text-sm font-bold">{t("current_location")}:</p>
-                <p className="text-emphasis mb-2 ml-1 text-sm">
-                  {getHumanReadableLocationValue(booking.location, t)}
-                </p>
-              </>
-            )}
+            <p className="text-emphasis mb-2 ml-1 mt-6 text-sm font-bold">{t("current_location")}:</p>
+            <p className="text-emphasis mb-2 ml-1 text-sm">
+              {getHumanReadableLocationValue(booking.location, t)}
+            </p>
             <Form
               form={locationFormMethods}
               handleSubmit={async (values) => {
-                const { locationType: newLocation, displayLocationPublicly } = values;
-
-                let details = {};
-                if (newLocation === LocationType.InPerson) {
-                  details = {
-                    address: values.locationAddress,
-                  };
-                }
-                const eventLocationType = getEventLocationType(newLocation);
-
-                // TODO: There can be a property that tells if it is to be saved in `link`
-                if (
-                  newLocation === LocationType.Link ||
-                  (!eventLocationType?.default && eventLocationType?.linkType === "static")
-                ) {
-                  details = { link: values.locationLink };
-                }
-
-                if (newLocation === LocationType.UserPhone) {
-                  details = { hostPhoneNumber: values.locationPhoneNumber };
-                }
-
+                const { locationType: newLocationType } = values;
+                let newLocation;
                 if (eventLocationType?.organizerInputType) {
-                  details = {
-                    ...details,
-                    displayLocationPublicly,
-                  };
+                  newLocation = values.locationAddress || values.locationLink || values.locationPhoneNumber;
+                } else {
+                  newLocation = newLocationType;
                 }
 
-                if (values.credentialId) {
-                  details = {
```

Comment:
We don't need all this logic. Requirement is to just send what is set out of the three options